### DISCUSSION TITLE
Customer Sign up method

### DISCUSSION THREAD

**Discussion by nothingmuch1** [phplime/Qrexorder#10] [Ideas]
Qrexorder is the most amazing php script I have seen so far on Codecanyon, and here is a request to improve it further. Can you please change the customer sign up method from phone number to email, so that customers can reset their passwords using that email, because it is not always easy to remember the security question, and then customers need to take help of restaurants to reset their passwords. 

Please include this feature in the upcoming update.
Thanks.

**Comment by nothingmuch1** [comment]
So that the customer can sign in using email and password, and not phone............... in this method, PHPMailer can be used to reset the password. Please consider it. I am facing issues with customer password reset.

**Reply by phplime** [reply]
I have a question here, if you do the registration with email, what happens when you need to send a Whatsapp message or WhatsApp order?   so there will be email and phone both in registration, right?

**Reply by nothingmuch1** [reply]
But when a customer forgets the password, and needs to reset it, he should either remember the security question, or he should contact the restaurant to reset it.

The answer is that, we can include both phone and email during customer registration, but we can use email based as primary method to reset the password, not the security question. It already exists for restuarants, such as when a restaurant owner forgets the password, he uses the email to reset the password, and not phone number. I hope that same can be done for the customers too.

**Reply by nothingmuch1** [reply]
Here is the example of Restro Saas script, pretty flawed script, and Qrexorder is miles ahead from that script, but we can look into the way they handle cusotmer signup and signin as well as the forget password.

Customer Registration Page: https://restro.infotechgravity.com/chicken-store/register
Customer Login Page: https://restro.infotechgravity.com/chicken-store/login
Forgot Password Page: https://restro.infotechgravity.com/chicken-store/forgotpassword

You will see that they are capturing the phone number, but they are using the email to login and reset the password for the customers.

**Reply by nothingmuch1** [reply]
So yes, we need both phone and email in registration, but email [and not phone] should be used to login and reset the password for customers.
